app/api/workday/route.ts

```diff
@@ -0,0 +1,12 @@
+import { getWorkdayData } from "@/app/about/queries"
+
+interface WorkdayRequest extends Request {}
+
+type WorkdayResponse = Response
+
+export async function GET(req: WorkdayRequest): Promise<WorkdayResponse> {
```

Comment:
I believe this could just be this

```suggestion
export async function GET(req: Request): Promise<Response> {
```